src/components/SmartMeterReadingsChart.scss

```diff
@@ -0,0 +1,28 @@
+.smartMeterReadingsChart {
+    & .button-switcher {
+        box-shadow: 0 2px 10px rgba(0, 0, 0, 0.267125);
+    
+        button {
+            background: #654784;
+            color: white;
+            font-size: 14px;
+    
+            &:hover, &.selected {
+                background: #593e74;
```

Comment:
Please use SCSS variables to support easier theming in future (as we already have this need in Origin)